Makefile.defs

```diff
@@ -445,8 +445,14 @@ ifeq "$(CC_IS_CLANG)" "0"
 else
 	# NOTE: Clang's default std value is... iffy. It appears to behave on Linux, but not on Darwin.
 	#       https://clang.llvm.org/cxx_status.html
-	CSTD_FLAGS:=-std=c11
-	CXXSTD_FLAGS:=-std=c++11
+	ifdef DARWIN
+		CSTD_FLAGS:=-std=c11
+		CXXSTD_FLAGS:=-std=c++11
+	else
+		# Keep GNU extensions enabled on Linux (mainly because the CI's Clang version sucks so very hard).
```

Comment:
To be clear, that's Clang 8, the latest - 1. I'll probably update it to 9 sometime. Not some curious archeological find from the depths of Ubuntu 16.04. :-P

Reply:
Are you sure it's Clang 8? o_O.

FWIW, luajit builds just fine w/ c11 on Linux w/ Clang 9, and macOS w/ Xcode 11. -_-".

Reply:
Unless Ubuntu mangles LLVM/Clang beyond recognition, which wouldn't surprise me all that much ;).

Reply:
Quite sure:

```
$ docker run koreader/kobase-clang:0.1.2 clang --version
clang version 8.0.1-svn363027-1~exp1~20190611212422.76 (branches/release_80)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

Also: #944

Reply:
Doesn't matter if Ubuntu does or doesn't; it comes straight from the horse's mouth:

https://github.com/koreader/virdevenv/blob/aea69d62b158dbfbf925e8fa22757848861d5d85/docker/ubuntu/baseimage-clang/clang8.list

Reply:
*sadface*

Reply:
https://github.com/koreader/virdevenv/pull/45 is ready to go